player/src/androidTest/kotlin/com/tidal/sdk/player/playlog/PlayLogTest.kt

```diff
@@ -129,6 +132,14 @@ class PlayLogTest {
         val job = launch { player.playbackEngine.events.first { it is Event.Release } }
         player.release()
         job.join()
+        verify(eventSender, atMost(Int.MAX_VALUE))
```

Comment:
Just to understand: This is added to ensure that we verify eventSender.sendEvent calls in each test? If that is true, is that really necessary? It sounds a bit like a test of a test 😅

Reply:
This needs to be looked at in combination with the verifyNoMoreInteractions call below.

The intention is to prevent unverified eventSender.sendEvent calls _for playlog events specifically_ from going unverified, thus opening the door to for example scenarios that send replicated events but their test passes even though it only verifies one call.

The reason for this strange way of approaching the situation is that there's no verifyNoMoreInteractions(argumentMatcher)-like call, so I do that by combining first this call, which gets any non-playlog related calls, and then the one after, which at that point becomes specific to playlog calls since all others have already been verified.

Reply:
Alright, thanks for explaining 👌